containers/codespaces-linux/.devcontainer/Dockerfile

```diff
@@ -33,32 +33,38 @@ ADD symlinkDotNetCore.sh /tmp/codespace/symlinkDotNetCore.sh
 
 ADD git-ed.sh /tmp/codespace/git-ed.sh
 
- # Install packages, setup codespace user
+# Install packages, setup codespace user
 RUN apt-get update -yq\
     && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -yq install --no-install-recommends apt-utils dialog 2>&1 \
     && apt-get install -yq \
-        default-jdk \
-        vim \
-        sudo \
-        xtail \
-        fish \
-        zsh \
-        curl \
-        gnupg \
-        apt-transport-https \
-        lsb-release \
-        software-properties-common \
-        unzip \
+    default-jdk \
+    vim \
+    sudo \
+    xtail \
+    fish \
+    zsh \
+    curl \
+    gnupg \
+    apt-transport-https \
+    lsb-release \
+    software-properties-common \
+    unzip \
     #
     # Be sure git is up to date due to DSA 4659-1, DSA 4657-1
     && apt-get upgrade -yq git \
     #
+    # Install Git CLI v 0.11.0
+    && echo "Downloading github CLI" \
+    && curl --progress-bar -OL https://github.com/cli/cli/releases/download/v0.11.0/gh_0.11.0_linux_amd64.deb \
+    && echo "Installing github CLI" \
+    && sudo apt install ./gh_0.11.0_linux_amd64.deb \
```

Comment:
I don't think we need to run `sudo` since we are already running as `root` here.